drivers/espi/espi_mchp_xec.c

```diff
@@ -583,15 +581,15 @@ static int espi_xec_flash_read(const struct device *dev,
 		return -EIO;
 	}
 
-	if (pckt->len > MAX_SLAVE_BUFFER_SIZE) {
+	if (pckt->len > CONFIG_ESPI_FLASH_BUFFER_SIZE) {
```

Comment:
same here

Reply:
No. If the channel are disabled it's not possible to reach this point